build/tools/docker/ubuntu.18.04/Dockerfile

```diff
@@ -29,7 +30,8 @@ RUN apt-get update && \
         lldb-3.6 \
         wget && \
```

Comment:
If this long list can be pared down because the Ubuntu 18.04 image contains them already, I'll do that in #30729